File: challenges/6-converter-segundos/python/ellenmariadev/converter_segundos.py
segundos = int(input('Segundos: '))

hora = segundos // 3600
minuto = (segundos - hora * 3600) // 60
segundo = segundos - hora * 3600 - minuto * 60

print(f'{hora:02d}:{minuto:02d}:{segundo:02d}')

File: challenges/6-converter-segundos/java/herco/ConverterSegundos.java
import java.util.Scanner;

public class ConverterSegundos {
    static final int HORA = 3600;
    static final int MINUTO = 60;

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Entre o valor em segundos: ");
        int entradaEmSegundos = scanner.nextInt();

        scanner.close();

        System.out.println(formataHora(entradaEmSegundos / HORA)
                   + ":" + formataHora((entradaEmSegundos % HORA) / MINUTO)
                   + ":" + formataHora((entradaEmSegundos % HORA) % MINUTO));
    }

    private static String formataHora(int valorParaFormatar){
        return valorParaFormatar < 10 ? "0" + valorParaFormatar : String.valueOf(valorParaFormatar);
    }
}